src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -39,5 +39,17 @@ public void periodic() {
                 m_intake.set(0); break;
         }
     }
+    private static class Intake_Speed {
+        public int intake = 0;
+        public Intake_Speed(int intake){
+            this.intake = intake;
+        }
+    }
+
+    private final Intake_Speed Cone_In = new Intake_Speed(0);
+    private final Intake_Speed Cone_Out = new Intake_Speed(0);
+    private final Intake_Speed Cube_In = new Intake_Speed(0);
+    private final Intake_Speed Cube_Out = new Intake_Speed(0);
+    private final Intake_Speed Stopped = new Intake_Speed(0);
```

Comment:
What are you trying to do here? If you're trying to have a set of constant intake speeds, put it in RPMs in `Constants.java`.